wit-manifest.json

```diff
@@ -1,237 +1,247 @@
 [
     {
+        "source": "git@github.com:sifive/freedom-devicetree-tools.git",
         "commit": "fd45cf28ecc6773699219b01663b7e55e1dbc4f9",
-        "name": "freedom-devicetree-tools",
-        "source": "git@github.com:sifive/freedom-devicetree-tools.git"
+        "name": "freedom-devicetree-tools"
```

Comment:
Why are these being rearranged? This will make rebasing/cherry-picking/reverting/merging messy.

Reply:
I am not sure either as I did not anything special. I am hoping when I update the work space to pickup the approved metal, this will revert back. But won't merge until I know why. I agree with your concern